Bookish/Bookish.Web/Controllers/HomeController.cs

```diff
@@ -29,6 +29,33 @@ public IActionResult BookPage()
             List<BookInfo> books = (List<BookInfo>)db.Query<BookInfo>("select * from BookInfo");
             return View(books);
         }
+        public IActionResult BookInfo(int ISBN)
+        {
+            IDbConnection db = new SqlConnection("Server = localhost; Database = Bookish; Integrated Security = True; MultipleActiveResultSets = true;");
+            
+            //string sql = "select * from Books WHERE ISBN = " + ISBN;
+            //List<Book> books = (List<Book>)db.Query<Book>(sql);
+
+            string sql2 = "select BookName from BookINFO WHERE ISBN = " + ISBN;
+            string name = db.Query<string>(sql2).Single();
+
+            string sql3 = "select count(*) from Books WHERE ISBN = " + ISBN;
+            int numberofcopies = db.Query<int>(sql3).Single();
+
+            string sql4 = "SELECT count(books.id) from books, borrowed where books.id = borrowed.bookid and books.ISBN = " + ISBN;
+            int numberoftaken = db.Query<int>(sql4).Single();
+            // SELECT books.id from books, borrowed where book.id = borrowed.bookid
+            //Select Accounts.accountname, book.id,borrowed.duedate from books, borrowed, accounts where books.id = borrowed.bookid and borrowed.userid = Accounts.id and books.ISBN =
+            string sql5 = "Select Accounts.accountname, books.id,borrowed.duedate from books, borrowed, accounts where books.id = borrowed.bookid and borrowed.userid = Accounts.id and books.ISBN = " +ISBN;
```

Comment:
Instead of sql2, sql3, etc. I would either: 1) use a single variable e.g. queryString for all of them, or 2) put them directly into the Query() calls without using variables.